src/topology/bounded_continuous_function.lean

```diff
@@ -466,7 +466,7 @@ sub_le_iff_le_add'.1 $ (abs_le.1 $ @dist_coe_le_dist _ _ _ _ f g x).2
 end normed_group
 
 section normed_space
-/-! In this section, if `β` is a normed space, then we show that the space of bounded
+/- In this section, if `β` is a normed space, then we show that the space of bounded
```

Comment:
Currently we use the following style for per-section docstrings. This style was introduced after this file was initially written. These comments will show on the documentation website once handling of module docstrings is fixed in core lean. 
```suggestion
/-!
### Normed space structure

In this section, if `β` is a normed space, then we show that the space of bounded
```